admin/products_price_manager.php

```diff
@@ -1088,7 +1088,9 @@ function updateSpecialsNet() {
               </div>
             </div>
           <?php } else { ?>
+            <?php if (sizeof($discount_name) == 0) { ?> 

```

Comment:
Shouldn't we be using `count` instead of `sizeof`?

Reply:
we should...